articles/nginx-reverse-proxy/index.md

```diff
@@ -0,0 +1,147 @@
+# Deploying Multiple Applications to VM with NGINX as a Reverse Proxy
+
+## Introduction
+Ever wondered how more than one application is being deployed to the same machine, and how traffic is being routed to the corresponding applications?
+Keep reading to know all about it!
+## Goals
+By the end of the article, one would be able to understand
```

Comment:
article, you'll understand would flow better.

Reply:
Sorry, @HarishTeens. I meant replace with "article, you'll understand" the would flow better part was purely me explaining what it would improve.

Reply:
> Sorry, @HarishTeens. I meant replace with "article, you'll understand" the would flow better part was purely me explaining what it would improve.

Not really sure what you meant to say, this is how the line is now "By the end of the article, you'll understand would flow better."

Reply:
Consider changing line 7 to "By the end of the article, you'll understand", @HarishTeens.

Reply:
Sorry for that, the above line doesn't make any sense. I've changed it as suggested.